agent/runner/actions/postgresql_query_show_action_test.go

```diff
@@ -49,13 +49,13 @@ func TestPostgreSQLQueryShow(t *testing.T) {
 		b, err := a.Run(ctx)
 		require.NoError(t, err)
 		assert.LessOrEqual(t, 22000, len(b))
-		assert.LessOrEqual(t, len(b), 33668)
+		assert.LessOrEqual(t, len(b), 37989)
```

Comment:
These kind of tests seems useless for me. Not sure what it can catch.

Reply:
agree